.github/workflows/ci.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Check goimports
         if: ${{ always() }}
         run: |
-          go install golang.org/x/tools/cmd/goimports
+          go install golang.org/x/tools/cmd/goimports@latest
```

Comment:
I'm always a little wary of `@latest`, since builds would not longer be strictly reproducible.  (Not that this _wasn't_ an issue with the bare `go get`s.)

Is there any significant downside to using the fixed sha referenced in the go.mod?

Ditto the other two.

Reply:
Agreed. It is safer to use a specific version rather than the lastest one.